scripts/generator/graphql_generator/csharp/Cart.cs.erb

```diff
@@ -159,6 +161,21 @@ namespace <%= namespace %> {
         }
 
         /// <summary>
+        /// Launches an external browser with the Cart's web checkout link loaded. This can be used to perform
+        /// a cart checkout when no native web view is available, such as on desktop, or from the Unity editor.
+        /// </summary>
+        /// <param name="success">called when the game has regained focus and the checkout was successful.</param>
+        /// <param name="cancelled">called when the game has regained focus but the checkout has not completed.</param>
+        /// <param name="failure">called when the game has regained focused and an error has occured during checkout.</param>
+        public void CheckoutWithExternalBrowser(CheckoutSuccessCallback success, CheckoutCancelCallback cancelled, CheckoutFailureCallback failure) {
```

Comment:
As per the discussion with @mikkoh, I believe the best is `Option 2`. This keeps the intent of the methods clear. The amount of work to do `Option 1` seems the same as `Option 2` so might as well do `Option 2`